src/stores/UI.ts

```diff
@@ -0,0 +1,30 @@
+import { AsyncStorage } from 'react-native';
+import { types, flow, applySnapshot, onSnapshot } from 'mobx-state-tree';
+import { config } from '~config';
+
+export const UI = types.model('UI', {
+    componentId: types.optional(types.string, ''),
+    isBeta: types.optional(types.boolean, config.isTestFlight || false),
+  })
+  .actions(self => ({
+    setComponentId(componentId?: string): void {
+      if (componentId) {
+        self.componentId = componentId;
+      }
+    },
+    setIsBeta(isBeta: boolean): void {
+      self.isBeta = isBeta;
+    },
+    hydrate: flow(function* (): IterableIterator<Promise<string | null>> {
```

Comment:
I'm not sure I haven't try, but is the notation `Array<thing>` is not firing an error?

Reply:
I tried that and got error